### PR TITLE
Add support for logo dimensions

### PR DESCRIPTION
**Pull request by benjaminwols**

### CODE REVIEW

**Comment by benjaminwols** [comment]
[sc-51796]

File: lib/onelogin/ruby-saml/settings.rb [hunk 1]
```diff
@@ -71,6 +71,8 @@ def initialize(overrides = {}, keep_security_attributes = false)
       attr_accessor :description
       attr_accessor :privacy_statement_url
       attr_accessor :logo
+      attr_accessor :logo_width
+      attr_accessor :logo_height
```
Comment (FelixdeJonge) [inline lib/onelogin/ruby-saml/settings.rb:75]: Do we want this change covered in the settings test as well? There's quite a few accessors that aren't covered already.

**Review by FelixdeJonge — CHANGES_REQUESTED** [review]

File: lib/onelogin/ruby-saml/settings.rb (hunk 1 above)
Comment (benjaminwols) [reply lib/onelogin/ruby-saml/settings.rb:75]: Don't think it is necessary

**Review by FelixdeJonge — APPROVED** [review]

**Comment by ddanco** [comment]
@benjaminwols it looks like Felix's approval wasn't enough to merge this, and I don't see the Settings visible, so I'm guessing I don't either. is this something you'll have to push through? 

Also, are those failing unit tests anything we need to worry about, or is that all fine?

**Pull request merged**